run_all.py

```diff
@@ -0,0 +1,39 @@
+# run with `python3 ../run_all.py`  (must be in build dir)
```

Comment:
Use a shebang instead

Reply:
What is a shebang?

Reply:
https://en.wikipedia.org/wiki/Shebang_(Unix)

Basically, it's a directive to the OS that tells it what program to use to execute a script.